src/fastoad_cs25/models/weight/mass_breakdown/cs25.py

```diff
@@ -34,6 +34,15 @@ class Loads(ExplicitComponent):
 
     """
 
+    def initialize(self):
+        self.options.declare(
+            "fuel_load_alleviation",
+            types=bool,
+            default=True,
+            desc="If False this simulates a dry wing,"
+            "i.d. the sizing load 2 does not take into account the fuel weight.",
```

Comment:
"i.d." ? Did you mean "i.e." ?

Reply:
Indeed, fixed in [6babc](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/38/commits/6babcfa6e13b1e259a36fcfe775c8f4d1e9719bd)